CRM/Financial/Form/BatchTransaction.php

```diff
@@ -76,12 +86,12 @@ public function preProcess() {
    * Build the form object.
    */
   public function buildQuickForm() {
-    if ($this->_batchStatusId == 2) {
+    if ($this->_batchStatusId == $this->batchStatuses['Closed']) {
       $this->add('submit', 'export_batch', ts('Export Batch'));
     }
 
-    // do not build rest of form unless it is open batch
-    if ($this->_batchStatusId != 1) {
+    // do not build rest of form unless it is open/reopened batch
+    if (!in_array($this->_batchStatusId, array($this->batchStatuses['Open'], $this->batchStatuses['Reopened']))) {
```

Comment:
best idea would be to check
if (!in_array($this->_batchStatus, array('Open', 'Reopened'))) {